src/components/Switch/Switch.tsx

```diff
@@ -0,0 +1,61 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { Size } from "@utilities/enum";
+import generateRandomId from "@utilities/generateRandomId";
+import { MouseEvent, ReactElement } from "react";
+import Label from "../Form/Label/Label";
+import css from "./Switch.module.css";
+
+export type SwitchProps = {
+    on?: boolean;
+    disabled?: boolean;
+    label?: string;
+    name?: string;
+    size?: Size.Large | Size.Small;
+    onChange?: (e: MouseEvent) => void;
+};
+
+const merge = (classNames: (string | undefined | boolean)[]) => classNames.filter(Boolean).join(" ");
```

Comment:
nitty-gritty helper. how about extract it to a utility file and reuse it in the other please we do similar stuff? cc @SamuelAlev

Reply:
Really great idea, remind me of [classnames](https://www.npmjs.com/package/classnames)

Reply:
alright, done here: https://github.com/Frontify/arcade/pull/8/commits/cefe40c2b5089d3d87ebf13018faab91ebbfd61c